drivers/platform/generic/timer.c

```diff
@@ -0,0 +1,184 @@
+/***************************************************************************//**
+*   @file   timer.c
+*   @brief  Timer and GIC control module source.
```

Comment:
So far, this is just a Timer control driver template.
I think it would make sense to add GIC text in the file description only when some sort of GIC functionality will be implemented in timer.c/.h.

Reply:
True, this text is a mistake from a previous implementation. Will remove GIC mentions.